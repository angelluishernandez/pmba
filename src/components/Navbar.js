import React from "react";
import { NavbarWrapper } from "./styled-components/NavbarWrapper";

const Navbar = () => {
	return <NavbarWrapper>This is the navbar</NavbarWrapper>;
};

export default Navbar;
