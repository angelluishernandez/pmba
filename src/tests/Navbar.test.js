import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJSON from "enzyme-to-json";
import React from "react";

import Navbar from "../components/Navbar";

configure({ adapter: new Adapter() });

test("should render navbar correctly", () => {
	const wrapper = shallow(<Navbar />);
	expect(toJSON(wrapper)).toMatchSnapshot();
});
