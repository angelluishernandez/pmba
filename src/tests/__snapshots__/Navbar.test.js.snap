// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render navbar correctly 1`] = `
<styled.nav>
  <Link
    to="/"
  >
    The Power MBA
  </Link>
</styled.nav>
`;
